src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -708,11 +717,10 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
           <Button
             mx={-2}
             size='lg'
-            colorScheme={isTradingActive === false ? 'red' : 'blue'}
+            colorScheme={isTradingActive === false || status === TxStatus.Failed ? 'red' : 'blue'}
             onClick={handleSignTx}
-            isDisabled={isTradingActive === false}
+            isDisabled={isTradingActive === false || status === TxStatus.Failed}
             isLoading={
-              status === TxStatus.Pending ||
```

Comment:
@kaladinlight this is the root cause of the regression, we want to bring this back

Reply:
1a323633b0